doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-con.adoc

```diff
@@ -0,0 +1,7 @@
+[id='business-central-security-management-con_{context}']
+
+= Security management
+
+Security management is the process of managing users, groups, and permissions. You can control the access to {CENTRAL} resources and features from the {CENTRAL} Security management page.
```

Comment:
You can control access to {CENTRAL} resources

Reply:
Done.